pkg/deploy/api/v1/register.go

```diff
@@ -9,9 +9,11 @@ func init() {
 		&DeploymentConfig{},
 		&DeploymentConfigList{},
 		&DeploymentConfigRollback{},
+		&Scale{},
 	)
 }
 
 func (*DeploymentConfig) IsAnAPIObject()         {}
 func (*DeploymentConfigList) IsAnAPIObject()     {}
 func (*DeploymentConfigRollback) IsAnAPIObject() {}
+func (*Scale) IsAnAPIObject() {}
```

Comment:
Upstream didn't do that. They just call it scale, and it's a subresource under both ReplicationController and Deployment.